src/domain/Uuid.test.ts

```diff
@@ -0,0 +1,14 @@
+import { Uuid } from './Uuid';
```

Comment:
Why not just export the `v4` function from the `UUID` domain? Actual reference equality is quite nice

Reply:
Ah you want to extend it